ioos_catalog/models/dataset.py

```diff
@@ -39,6 +40,8 @@ class Dataset(BaseDocument):
                 'keywords'          : [unicode],  # Search keywords
                 'variables'         : [unicode],  # Environmental properties measured by this dataset
                 'asset_type'        : unicode,    # See the IOOS vocablary for assets: http://mmisw.org/orr/#http://mmisw.org/ont/ioos/platform
+                'time_min': datetime, # start time of this service
+                'time_max': datetime, # end time of this service
```

Comment:
> One of Guido's key insights is that code is read much more often than it is
> written. The guidelines provided here are intended to improve the readability
> of code and make it consistent across the wide spectrum of Python code. As PEP
> 20 says, "Readability counts".


> Some other good reasons to ignore a particular guideline:
> 
> 1. When applying the guideline would make the code less readable, even for
>    someone who is used to reading code that follows this PEP.
> 2. To be consistent with surrounding code that also breaks it (maybe for
>    historic reasons) -- although this is also an opportunity to clean up someone
>    else's mess (in true XP style).
> 3. Because the code in question predates the introduction of the guideline and
>    there is no other reason to be modifying that code.
> 4. When the code needs to remain compatible with older versions of Python that
>    don't support the feature recommended by the style guide.